src/repl.zig

```diff
@@ -832,9 +862,41 @@ pub fn ReplType(comptime MessageBus: type) type {
                         }
                     }
                 },
+                .get_account_transfers => {
+                    const get_account_transfers_results = std.mem.bytesAsSlice(
+                        tb.Transfer,
+                        result,
+                    );
+
+                    if (get_account_transfers_results.len == 0) {
```

Comment:
Hm, but we _should_ get no results if the account wasn't transacted against?

Reply:
likewise in lookup_accounts and lookup_transfers! I'm going to drop the word "Failed" then.